checks/macos.py

```diff
@@ -66,3 +66,28 @@ def checkRosettaTranslationStatus(lines):
     if (len(search('Rosetta translation used: true', lines)) > 0):
         return [LEVEL_WARNING, "Intel OBS on Apple Silicon Mac",
                 "You are running the Intel version of OBS on an Apple Silicon Mac. You may get improved performance using the Apple Silicon version of OBS."]
+
+
+def checkMacPermissions(lines):
+    macPerms = search('[macOS] Permission for', lines)
+    deniedPermissions = []
+
+    for line in macPerms:
+        if 'denied' in line:
+            found = re.search(r'Permission for (.+) denied', line)
+            if found:
+                permissionName = found.group(1).title()
+                permissionDescription = {
+                    "Audio Device Access": "<b>Microphone</b>: OBS requires this permission if you want to capture your microphone.",
+                    "Video Device Access": "<b>Camera</b>: This permission is needed in order to capture content from a webcam or capture card.",
+                    "Accessibility": "<b>Accessibility</b>: For keyboard shortcuts (hotkeys) to work while other apps are focused this permission is needed.",
+                    "Screen Capture": "<b>Screen Recording</b>: OBS requires this permission to be able to capture your screen."
```

Comment:
Keep the wording consistent. Use active voice.
```
OBS requires this permission [for X feature | to do X action]
```
or
```
OBS requires [Permission Name] [...]
```

Reply:
I can still adjust if you'd like but this language was taken directly from what is already written in OBS.

<img width="687" alt="Screenshot 2024-03-14 at 11 12 35 AM" src="https://github.com/obsproject/loganalyzer/assets/86465454/a5bdcc7c-4d8b-4459-9add-26ae97807e68">

Reply:
I believe this wording might come from me (although I may have been advised by others?), if there are better suggestions we can change it in OBS itself as well.

Reply:
I did check the blame and you originally had all the strings consistently worded, but changed them based on feedback and I did not review that PR.

Reply:
For the sake of getting this check applied to the analyzer, we can keep the strings consistent with OBS for now. I would like to revisit the strings in OBS in the future, and then also revisit them here.